.github/workflows/build-and-test.yml

```diff
@@ -30,16 +35,33 @@ jobs:
       - name: Run node server side tests
         run: yarn test:serverside
 
+  run-checks:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
```

Comment:
@suzubara should we use 12 here, or default to a newer release if we're intending to support up to 16?

Reply:
IMO in the interest of changing one thing at a time, we could bump this to 16 in https://github.com/trussworks/react-uswds/pull/1778

Reply:
Ooof. I made these so long ago, I forgot I had that other PR to make that change 😆 Thanks!

Reply:
Ooof. I made these so long ago, I forgot I had that other PR to make that change 😆 Thanks!